components/places/src/storage/mod.rs

```diff
@@ -199,7 +199,7 @@ pub fn run_maintenance(conn: &PlacesDb) -> Result<()> {
 
 pub(crate) fn put_meta(db: &PlacesDb, key: &str, value: &ToSql) -> Result<()> {
     db.execute_named_cached(
-        "REPLACE INTO moz_meta (key, value) VALUES (:key, :value)",
+        "INSERT OR REPLACE INTO moz_meta (key, value) VALUES (:key, :value)",
```

Comment:
Hmm, I thought these were equivalent, but maybe not! Was this for style, or did you notice `INSERT OR REPLACE` behaving differently than `REPLACE`?

Reply:
> Hmm, I thought these were equivalent, but maybe not! Was this for style, or did you notice `INSERT OR REPLACE` behaving differently than `REPLACE`?

Actually, that was my screwup - I think what happened is that I started the patch when it was just `INSERT INTO`, I changed it to `INSERT OR REPLACE INTO` while it was otherwise changed to just `REPLACE INTO`, and I ended up accidentally keeping mine when merging. I reverted that change.